sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/KnownManagedRuleSet.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.network.models;
+
+/**
+ * Known managed rule set.
+ */
+public enum KnownManagedRuleSet {
```

Comment:
Since it under network namespace. Wondering whether it better to be `KnownWebApplicationFirewallManagedRuleSet`? Maybe too long? `KnownWafManagedRuleSet` seems strange.

Reply:
Maybe `KnownFirewallPolicyManagedRuleSet`?

Reply:
Network got a `AzureFirewallInner`. I assume they are different firewall.

Reply:
Yes, they are different. 
Other candidates maybe `KnownWebApplicationFirewallPolicyManagedRuleSet`, `KnownWafPolicyManagedRuleSet`..

Guess `KnownWebApplicationFirewallManagedRuleSet` then..